frontend/integration-tests/protractor.conf.ts

```diff
@@ -71,6 +71,9 @@ const testSuites = {
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts',
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts',
   ]),
+  kubevirtPlugin: suite([
```

Comment:
we can put this in our plugin kubevirt-plugin package.json, for example as kubevirt-gating

Reply:
done.